moto/cognitoidp/models.py

```diff
@@ -900,6 +900,19 @@ def admin_update_user_attributes(self, user_pool_id, username, attributes):
         user = user_pool.users[username]
         user.update_attributes(attributes)
 
+    def admin_user_global_sign_out(self, user_pool_id, username):
+        user_pool = self.user_pools.get(user_pool_id)
+        if not user_pool:
+            raise ResourceNotFoundError(user_pool_id)
```

Comment:
Can we get unit tests for these two exceptions as well, please?

Reply:
No Problem - additional tests committed